orm/entities/SubmissionVersion/TallySheetVersion/TallySheetVersion_PRE_ALL_ISLAND_RESULTS_BY_ELECTORAL_DISTRICTS.py

```diff
@@ -14,6 +14,7 @@
 from orm.entities.Submission import TallySheet
 from orm.enums import TallySheetCodeEnum, AreaTypeEnum
 from sqlalchemy import and_
+from num2words import num2words
```

Comment:
@sm1990 
Add `num2words` to requirements.txt

Reply:
@dinukadesilva added to requirement.txt in https://github.com/ECLK/results-tabulation-api/pull/163